### PR TITLE
Add integration tests for several clients

### LINKED ISSUE #38: Add integration tests for the active contracts client
Build on top of #11, add integration tests for the active contracts client.

### LINKED ISSUE #39: Add integration tests for the command client
Build on top of #11, add integration tests for the command client.

### LINKED ISSUE #41: Add integration tests for the command submission client
Build on top of #11, add integration tests for the command submission client.

### LINKED ISSUE #43: Add integration tests for the package client
Build on top of #11, add integration tests for the package client.

### LINKED ISSUE #44: Add integration tests for the reset client
Build on top of #11, add integration tests for the reset client.

### LINKED ISSUE #45: Add integration tests for the time client
Build on top of #11, add integration tests for the time client.

### LINKED ISSUE #46: Add integration tests for the transaction client
Build on top of #11, add integration tests for the transaction client.

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Fixes #38
Fixes #39
Fixes #41
Fixes #43
Fixes #44
Fixes #45
Fixes #46

### CODE REVIEW

File: src/codec/CommandCodec.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const CommandCodec: Codec<PbCommand, Command> = {
         } else if (message.hasCreateandexercise()) {
             return CreateAndExerciseCommandCodec.deserialize(message.getCreateandexercise()!);
         } else {
-            throw new Error('Deserialization error, unable to discriminate value type - this is likely to be a bug');
+            throw new Error('Command deserialization error, unable to discriminate value type - this is likely to be a bug');
```
Comment (gerolf-da) [inline src/codec/CommandCodec.ts]: I assume this error is user-facing? One thing I liked about the error messages printed by node.js was that it told me where the bug likely occurred. All that is to say: is `this is likely a bug` referring to a user error in setting up the command, or a bug in the library?

Comment (stefanobaghino-da) [reply src/codec/CommandCodec.ts]: Very good point, I'll improve this error message to point out possible causes and solutions.

Comment (stefanobaghino-da) [reply src/codec/CommandCodec.ts]: Tracked [here](https://github.com/digital-asset/daml-js/issues/49).

Comment (stefanobaghino-da) [reply src/codec/CommandCodec.ts]: Should be taken care of, feel free to chime in and thanks for the feedback. :smiley:

**Pull request merged**